sci-biology/imagej/imagej-9999.ebuild

```diff
@@ -0,0 +1,131 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit git-r3 java-pkg-2 java-ant-2 desktop
+
+MY_PN="ij"
+IJ_PV="153" #plugins not currently available under 154
+
+DESCRIPTION="Image Processing and Analysis in Java"
+
+HOMEPAGE="
+	https://imagej.nih.gov/ij/
+	https://github.com/imagej
+"
+
+EGIT_REPO_URI="https://github.com/imagej/ImageJ"
+
+SRC_URI="
+	https://rsb.info.nih.gov/ij/images/ImageJ.png
+	plugins? ( https://wsr.imagej.net/distros/cross-platform/${MY_PN}${IJ_PV}.zip )"
+# plugins are under a different licenses and can be installed into user's $IJ_HOME/plugins
+#	plugins? ( http://rsb.info.nih.gov/ij/download/zips/${MY_PN}${IJ_PV}.zip )"
+
+KEYWORD="**"
```

Comment:
It's `KEYWORDS` not `KEYWORD`. But in any case live 9999 ebuilds should just have no KEYWORDS variable of have it set to an empty string.
```suggestion
```